contact_links/__manifest__.py

```diff
@@ -15,20 +15,22 @@
 v12.0.1.2	- Updated manifest and added Af-logo to description. \n
 			- Updated the security-groups in the datafile to match new version of af_security-groups. \n
 \n
+v12.0.1.4 - Made sure the contact links are computed with the customer number
 """,
     'author': 'Vertel AB',
     'license': 'AGPL-3',
     'website': 'http://www.vertel.se',
     'depends': [
         'contacts',
         'af_security',
-        'hr_360_view'
+        'hr_360_view',
+        'partner_view_360',
     ],
     'data': [
         'security/security.xml',
         'views/res_partner_view.xml',
         'security/ir.model.access.csv',
-        'data/server_action.xml',
+        # 'data/server_action.xml',
```

Comment:
Why just comment out this?
Should it be removed? And if so, then the file data/server_action.xml should be removed as well.

Reply:
That was commented out because its not going to be needed, but I left this here, if it case it will be needed after testing what I did.